ansible/roles/cyhy_dashboard/meta/main.yml

```diff
@@ -15,4 +15,6 @@ galaxy_info:
         - buster
   role_name: cyhy_dashboard
 
+collections:
+  - community.general
```

Comment:
Why is this `collections` block only in this particular `meta/main.yml` file?

Reply:
I only added `collections` blocks to the metadata files of the two roles that use non-builtin functionality. This role (`cyhy_dashboard`) uses `community.general.make` and the `mongo` roles has it because it uses `community.mongodb.mongodb_user`. I'm happy to add empty `collections: []` entries to all of the other roles for consistency.

Reply:
I'm good with adding empty collections entries for consistency.  @jsf9k, you good with that?

Reply:
> I'm good with adding empty collections entries for consistency. @jsf9k, you good with that?

Yes, I am good with this.  We should also add it to [`cisagov/skeleton-ansible-role`](https://github.com/cisagov/skeleton-ansible-role).

Reply:
So coming back here I actually found official documentation that had me remove these entries. Per the [documentation](https://docs.ansible.com/ansible/latest/collections_guide/collections_using_playbooks.html#using-collections-in-roles) that keyword is only used for namespace searching and is not involved in dependency resolution. Per https://github.com/ansible/ansible/issues/76030#issuecomment-942520399 there is no way for a role to depend on a collection as part of role metadata. The only way around this would be converting these Ansible roles into a collection and defining any collection dependencies there.

Reply:
Even better- thanks for looking into that. 👍